### PR TITLE
Adds some QoL changes to Custodian

### PR DESCRIPTION
**Pull request by Smokingsquirrel**
## About The Pull Request
This makes it possible to store the soap and space cleaner inside the neotheo utility belt.
Furthermore the cleansing carbine is now SLOT_BELT like other carbine type weapons. 
Slowdown from Custodian armor is removed to offset galoshes slowdown until i can figure out how to properly add NOSLIP to custodian shit, it doesnt work as of now.
The trash bag is now smaller but also takes in a smaller amount of trash to fit better in backpacks and dont take up too much space.

## Why It's Good For The Game
Custodian is currently very menial and quite the hassle to manage your resources properly. This small fix at least has you able to keep more equipment with you for the time being.

## Changelog
:cl:
tweak: Soap and space cleaner now fit the NeoTheo utility belt.
balance: trash bags are now smaller and better fit backpacks, but now also hold less trash.
balance: removed slowdown from custodian armor to make up on galoshes slowdown.
/:cl:

### CODE REVIEW

**Comment by Kurgis** [comment]
This increases slowdown, check parent objects.

**Comment by Smokingsquirrel** [comment]
> This increases slowdown, check parent objects.

thanks for the info, should be fixed now.

File: code/modules/projectiles/guns/matter/launcher/nt_sprayer.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	desc = "\"NeoTheology\" brand cleansing carbine. Uses solid biomass as ammo and dispense cleansing liquid on hit."
 	icon_state = "nt_sprayer"
 	icon = 'icons/obj/guns/matter/nt_sprayer.dmi'
-	slot_flags = SLOT_BACK
+	slot_flags = SLOT_BELT
```
[the remark this answers is not in the document]
Comment (Kurgis) [reply code/modules/projectiles/guns/matter/launcher/nt_sprayer.dm:6]: `SLOT_BACK | SLOT_BELT`

**Comment by Smokingsquirrel** [comment]
Ready for review

**Comment by Smokingsquirrel** [comment]
ignore this, apprently the merge with the other branch fucked my git AGAIN. Merge the other one, forget this and i will write this again when the other stuff is merged with master

**Pull request closed without merging**